src/BreathOfTheWild/Mods/MasterModeOptions/rules.txt

```diff
@@ -0,0 +1,111 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Master Mode Options"

```

Comment:
Wonder if a better name would be Master Mode Tweaks since it might imply that it modifies the existing master mode more and doesn't affect the normal options. That's just a thought though, choose what you think sounds better!